src/browser.ts

```diff
@@ -213,6 +248,7 @@ export class GrpcClient {
     const newServiceStub = service.create(serviceClientImpl, false, false);
     for (const methodName of methods) {
       newServiceStub[methodName] = (req, options, metadata, callback) => {
+        this.otherArgs[methodName] = options;
```

Comment:
Wait, this does not look right. These `options` are for this one current method call, they are not supposed to be saved in `this` (`this` is `GrpcClient`) since they will be used for other methods as well. I think options must not be saved between calls.

It's becoming really complicated here: seems like we need to pass some extra data all the way down to `serviceClientImpl` even though there is no 'official' way to do that (it only accepts request and callback).

Let's discuss it.